src/main/java/com/softwarewithpassion/Application.java

```diff
@@ -3,6 +3,7 @@
 public class Application {
 
     public static void main(String[] args) {
+        System.out.println("Hello Git! I am Alice.");
```

Comment:
Oh no, Alice, I would like my name to be first, can we please change the order of our names?